### DISCUSSION TITLE
focal ubuntu packages don't seem to be published but the documentation mentions them.

### DISCUSSION THREAD

**Discussion by EraYaN** [rabbitmq/rabbitmq-server#2776] [Other] [answered]
So PackageCloud and Bintray do not host focal (20.04, latest LTS from last year) packages but the install-debian documentation mentions the distribution.

Is this an error in the docs or in the package publishing? Or more: Is this an intentional omission?

**Comment by michaelklishin** [comment, marked as answer]
The docs mention Focal in the [context of Bintray](https://www.rabbitmq.com/install-debian.html#erlang-source-list-file).

However, this is a mere technicality. The package is identical for all distributions, we just have to specify a list of distribution in the publishing Concourse pipeline. You can specify `bionic` for distribution name on Ubuntu 20.04 or 20.10 and it should be perfectly fine.

We'll add `focal` as a distribution on PackageCloud, if anything because Bintray is going away by May 2021.
